janus_server/Cargo.toml

```diff
@@ -11,6 +11,7 @@ tokio-console = ["console-subscriber"]
 jaeger = ["tracing-opentelemetry", "opentelemetry-jaeger"]
 otlp = ["tracing-opentelemetry", "opentelemetry-otlp", "opentelemetry-semantic-conventions", "tonic"]
 prometheus = ["opentelemetry-prometheus", "dep:prometheus"]
+test-util = ["janus_core/test-util", "lazy_static", "testcontainers"]
```

Comment:
I don't think there's much risk to bumping MSRV to 1.60, since the Rust toolchain installed in GH actions runner is [already 1.61](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2204-Readme.md#rust-tools), and our container builds are already using `rust:1.62.0-alpine`.